z3c/dependencychecker/package.py

```diff
@@ -167,11 +167,29 @@ def top_level(self):
         )
 
         if os.path.exists(sources_top_level):
```

Comment:
This line succeeds if there's just one top level. If there are multiple, it fails because there's a `\n` in the string and another top level directory. But: that's a bit of a side effect.

a) Can it be generalized by just doing the `.split('\n')` right at the start? That will also work just fine if there is no `\n`.

b) In that case. you could also have the .py 'single_module_top_level' test inside the loop.

Reply:
You were right, and actually it was a side effect that it was working as I wrote it. Thanks for carefully reviewing it!

This new version uses, as you suggested, a for loop from the very beginning so all cases can be handled within it.